yt_dlp/extractor/youtube.py

```diff
@@ -3758,7 +3812,7 @@ def _get_requested_clients(self, url, smuggled_data):
                 excluded_clients.append(client[1:])
             elif client not in allowed_clients:
                 self.report_warning(f'Skipping unsupported client "{client}"')
-            elif client in self._BROKEN_CLIENTS.values():
+            elif client in self._BROKEN_CLIENTS:
                 broken_clients.append(client)
```

Comment:
this doesn't do anything anymore; should we check if the client has `REQUIRE_PO_TOKEN` and append it to `broken_clients` if there is no accompanying `po_token` passed?

Reply:
Hmm, we don't know about po_token at this point.. We would have to de-prioritize later 🤔

Reply:
Nothing addressed this, correct? Not saying we needed to, just confirming.

I do think we should remove this `broken_clients`/`BROKEN_CLIENTS` code since it is pointless and only making things more complicated than necessary

Reply:
> Hmm, we don't know about po_token at this point..

can't we call `_get_config_po_token` here?

Reply:
No, as the po token may not come from the config, and we need the data sync ID/visitor data and other runtime info